.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "astropy_helpers"]
```

Comment:
I'm pretty sure you need to keep this content around.

Reply:
Yes, seems to be what all the failing tests are complaining about. 😉 But trust me, I'm not _trying_ to remove this stuff -- just seems like every once in awhile, the astropy_helpers submodule wants to fall apart for no particular reason. 😞